package.json

```diff
@@ -18,16 +18,20 @@
     "react-router-dom": "^6.11.1"
   },
   "devDependencies": {
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^14.0.0",
     "@types/react": "^18.2.5",
     "@types/react-dom": "^18.2.4",
     "@vitejs/plugin-react": "^4.0.0",
     "autoprefixer": "^10.4.14",
+    "jsdom": "^22.0.0",
     "postcss": "^8.4.23",
     "tailwindcss": "^3.3.2",
     "vite": "^4.3.5",
-    "vitest": "^0.31.0",
-    "jsdom": "^22.0.0",
-    "@testing-library/react": "^14.0.0",
-    "@testing-library/jest-dom": "^5.16.5"
-  }
+    "vitest": "^0.31.0"
+  },
+  "description": "<h1 align=\"center\">\r  Gym Junkies 💫\r  </h1>",
+  "main": "postcss.config.js",
+  "author": "",
+  "license": "ISC"
```

Comment:
We're under MIT license and the `postcss.config.js` isn't the entry point for the project